pkg/dns/aws/dns.go

```diff
@@ -97,18 +96,6 @@ const (
 	deleteAction action = "DELETE"
 )
 
-func (p *Route53DNSProvider) AdjustEndpoints(dnsRecord *v1alpha1.DNSRecord, dnsPolicy *v1alpha1.DNSPolicy) error {
-	//dnsPolicy is not currently used, but will be once we add other routing strategies
-	totalIPs := 0
-	for _, e := range dnsRecord.Spec.Endpoints {
-		totalIPs += len(e.Targets)
-	}
-	for _, e := range dnsRecord.Spec.Endpoints {
-		e.SetProviderSpecific(ProviderSpecificWeight, awsEndpointWeight(totalIPs))
-	}
-	return nil
-}
-
```

Comment:
The original intention for `AdjustEndpoints` was that it would allow us to call out o provider specific logic to manipulate endpoints into a provider specific set. As we have progressed the geo work we now realise that it would be much better if the DNSRecord and endpoints are consistent regardless of DNS provider and instead any provider specific logic is dealt with as part of the `Ensure` call.